sql/moz-fx-data-marketing-prod/ga_derived/mobile_installs_adjust_v1/query.sql

```diff
@@ -0,0 +1,63 @@
+SELECT
+  date,
+  app,
+  network,
+  campaign,
+  adgroup,
+  creative,
+  CASE
+  WHEN
+    country = 'us'
+  THEN
+    'United States'
+  WHEN
+    country = 'de'
+  THEN
+    'Germany'
+  WHEN
+    country = 'it'
+  THEN
+    'Italy'
+  WHEN
+    country = 'uk'
+  THEN
+    'United Kingdom'
+  WHEN
+    country = 'ca'
+  THEN
+    'Canada'
+  WHEN
+    country = 'es'
+  THEN
+    'Spain'
+  WHEN
+    country = 'pl'
+  THEN
+    'Poland'
+  WHEN
+    country = 'fr'
+  THEN
+    'France'
+  ELSE
+    'Other'
+  END
+  AS country,
+  os,
+  device_type,
+  sum(clicks) AS clicks,
```

Comment:
Convention is to have built-in function names in uppercase.  This should be done automatically by `bqetl format`, but it looks like you branched off a pretty old version of the `main` branch, so I'd recommend merging `main` into your branch and then re-running `bqetl format`.